lumen/ai/llm.py

```diff
@@ -21,13 +24,19 @@ class Llm(param.Parameterized):
 
     use_logfire = param.Boolean(default=False)
 
+    interceptor_path = param.String(default=None)
```

Comment:
I would simply allow passing the interceptor instance, that way it's actually composable.